doc/scaling.rst

```diff
@@ -3,20 +3,29 @@ Scaling on Distributed Memory (Multiprocessing)
 ###############################################
 It's Easy
 ---------
-Only very minimal changes are needed to your daal4py code to allow it running on
-a cluster of workstations. Initialize the distribution engine::
+daal4py operates in SPMD-mode (Single Programm Multiple Data), which means your
+program is executed on several process (e.g. simialar to MPI). Any of the
+distributed algorithms return a the usable result on all processes. No worries,
+you do not need mpi4py or alike for this (even though of course you are free to
```

Comment:
I think it might be good to adjust this sentence to match the style of the other docs.  Maybe say "mpi4py is optional for the distributed algorithms and not required for SPMD-mode to work.